dbt/models/default/schema/default.vw_card_res_char.yml

```diff
@@ -109,7 +109,7 @@ models:
 
     tests:
       - unique_combination_of_columns:
-          name: vw_card_res_char_unique_by_pin_card_and_year
```

Comment:
All changes in `default/` are related to namespace changes. See PR body for more details.